complie error

**Describe the bug**
using apache/incubator-doris:build-env-1.2 to compile apache-doris-0.14.0-incubating
reported the following errors

[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/PaloInternalServiceVersion.java:[12,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/PaloInternalServiceVersion.java:[14,68] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[9,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[10,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[11,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[13,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[14,34] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[15,34] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[16,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[17,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[18,31] package org.apache.thrift.async does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[19,1] package org.apache.thrift.server.AbstractNonblockingServer does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[33,17] package org.slf4j does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[34,17] package org.slf4j does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[38,70] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[52,51] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[39,50] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[41,50] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[43,44] cannot find symbol
  symbol:   class IScheme
  location: class org.apache.doris.thrift.BatchSaveRowsetMetaResponse
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[43,54] cannot find symbol
  symbol:   class SchemeFactory
  location: class org.apache.doris.thrift.BatchSaveRowsetMetaResponse
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[9,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[10,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[11,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[13,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[14,34] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[15,34] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[16,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[17,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[18,31] package org.apache.thrift.async does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[19,1] package org.apache.thrift.server.AbstractNonblockingServer does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[33,17] package org.slf4j does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[34,17] package org.slf4j does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[38,65] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[52,51] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[111,63] package org.apache.thrift.meta_data does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[282,46] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[282,88] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[286,47] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[286,89] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[308,50] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[39,50] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[41,50] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[43,44] cannot find symbol
  symbol:   class IScheme
  location: class org.apache.doris.thrift.SaveRowsetMetaResponse
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[43,54] cannot find symbol
  symbol:   class SchemeFactory
  location: class org.apache.doris.thrift.SaveRowsetMetaResponse
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[9,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[10,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[11,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[13,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[14,34] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[15,34] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[16,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[17,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[18,31] package org.apache.thrift.async does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[19,1] package org.apache.thrift.server.AbstractNonblockingServer does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[33,17] package org.slf4j does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[34,17] package org.slf4j does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[38,50] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[58,51] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[111,63] package org.apache.thrift.meta_data does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[262,46] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[262,88] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[266,47] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[266,89] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[288,50] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[39,50] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[41,50] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[42,50] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[44,44] cannot find symbol
  symbol:   class IScheme
  location: class org.apache.doris.thrift.TStatus
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[44,54] cannot find symbol
  symbol:   class SchemeFactory
  location: class org.apache.doris.thrift.TStatus
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatusCode.java:[12,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatusCode.java:[14,53] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[124,63] package org.apache.thrift.meta_data does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[374,46] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[374,88] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[378,47] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[378,89] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TStatus.java:[408,50] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[329,84] cannot find symbol
  symbol:   class SchemeFactory
  location: class org.apache.doris.thrift.BatchSaveRowsetMetaResponse
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[335,74] cannot find symbol
  symbol:   class StandardScheme
  location: class org.apache.doris.thrift.BatchSaveRowsetMetaResponse
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[337,48] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[337,126] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[377,49] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[377,127] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[401,81] cannot find symbol
  symbol:   class SchemeFactory
  location: class org.apache.doris.thrift.BatchSaveRowsetMetaResponse
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[407,71] cannot find symbol
  symbol:   class TupleScheme
  location: class org.apache.doris.thrift.BatchSaveRowsetMetaResponse
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[410,49] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[410,126] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[429,48] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[429,125] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TAssertion.java:[12,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/TAssertion.java:[14,52] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveTabletMetaReq.java:[9,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveTabletMetaReq.java:[10,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveTabletMetaReq.java:[11,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveTabletMetaReq.java:[13,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveTabletMetaReq.java:[14,34] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveTabletMetaReq.java:[15,34] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveTabletMetaReq.java:[16,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveTabletMetaReq.java:[17,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveTabletMetaReq.java:[18,31] package org.apache.thrift.async does not exist
[INFO] 100 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] doris-fe-common 1.0.0 .............................. FAILURE [01:06 min]
[INFO] spark-dpp 1.0.0 .................................... SKIPPED
[INFO] fe-core 3.4.0 ...................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:07 min
[INFO] Finished at: 2021-06-21T12:48:32Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project doris-fe-common: Compilation failure: Compilation failure:
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/PaloInternalServiceVersion.java:[12,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/PaloInternalServiceVersion.java:[14,68] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[9,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[10,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[11,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[13,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[14,34] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[15,34] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[16,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[17,25] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[18,31] package org.apache.thrift.async does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[19,1] package org.apache.thrift.server.AbstractNonblockingServer does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[33,17] package org.slf4j does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[34,17] package org.slf4j does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[38,70] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[52,51] package org.apache.thrift does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[39,50] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[41,50] package org.apache.thrift.protocol does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[43,44] cannot find symbol
[ERROR]   symbol:   class IScheme
[ERROR]   location: class org.apache.doris.thrift.BatchSaveRowsetMetaResponse
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/BatchSaveRowsetMetaResponse.java:[43,54] cannot find symbol
[ERROR]   symbol:   class SchemeFactory
[ERROR]   location: class org.apache.doris.thrift.BatchSaveRowsetMetaResponse
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[9,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[10,32] package org.apache.thrift.scheme does not exist
[ERROR] /root/apache-doris-0.14.0-incubating/fe/fe-common/target/generated-sources/thrift/org/apache/doris/thrift/SaveRowsetMetaResponse.java:[11,32] package org.apache.thrift.scheme does not exist

--- a/docs/zh-CN/developer-guide/fe-idea-dev.md
+++ b/docs/zh-CN/developer-guide/fe-idea-dev.md
@@ -36,7 +36,7 @@ JDK1.8+, IntelliJ IDEA
 
 3. 如果仅进行fe开发而没有编译过thirdparty，则需要安装thrift，并将thrift 复制或者连接到 `thirdparty/installed/bin` 目录下
 
-    1. 安装 thrift 0.9.3 版本(注意：Doris基于 thrift 0.9.3 构建, 使用新版本会导致编译不通过)
+    1. 安装 thrift 0.9.3 版本(注意：Doris 基于 thrift 0.9.3 构建, 使用新版本会导致编译不通过)
 
         MacOS: `brew install thrift@0.9`
         Windows: `http://archive.apache.org/dist/thrift/0.9.3/thrift-0.9.3.exe`

--- a/docs/zh-CN/developer-guide/fe-idea-dev.md
+++ b/docs/zh-CN/developer-guide/fe-idea-dev.md
@@ -36,7 +36,7 @@ JDK1.8+, IntelliJ IDEA
 
 3. 如果仅进行fe开发而没有编译过thirdparty，则需要安装thrift，并将thrift 复制或者连接到 `thirdparty/installed/bin` 目录下
 
-    1. 安装 thrift 0.9.3 版本。
+    1. 安装 thrift 0.9.3 版本(注意：Doris基于 thrift 0.9.3 构建, 使用新版本会导致编译不通过)
 
         MacOS: `brew install thrift@0.9`
         Windows: `http://archive.apache.org/dist/thrift/0.9.3/thrift-0.9.3.exe`

